packages/hub/.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "next/core-web-vitals",
   "rules": {
+    "import/no-anonymous-default-export": "off",
```

Comment:
Note on this: I've recently decided that naming default components `page.tsx` and `layout.tsx` is a waste of time, so now I'm using `export default async function () { ... }`.